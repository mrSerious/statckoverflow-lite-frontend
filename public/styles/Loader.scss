.loader-overlay {
  background-color: $loader-overlay;
  color: $very-dark-grayish-blue;
  font-size: 2em;
  height: 100%;
  position: fixed;
  text-align: center;
  top: 0;
  width: 100%;
  z-index: 999999999999;

  .fas {
    -webkit-animation: fa-spin 3s infinite linear;
    animation: fa-spin 3s infinite linear;
    position: relative;
    top: 45%;
  }
}
