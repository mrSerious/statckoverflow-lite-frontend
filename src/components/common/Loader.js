import React from 'react';

const Loader = () => (
  <div className="loader-overlay">
    <i className="fas fa-circle-notch" />
  </div>
);

export default Loader;
