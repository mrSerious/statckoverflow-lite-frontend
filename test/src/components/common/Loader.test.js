import React from 'react';
import { shallow } from 'enzyme';

import Loader from '../../../../src/components/common/Loader';

test('App snapshot test', () => {
  const component = shallow(<Loader />);
  expect(component).toMatchSnapshot();
});
