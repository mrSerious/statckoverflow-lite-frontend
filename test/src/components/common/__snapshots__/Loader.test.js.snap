// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<div
  className="loader-overlay"
>
  <i
    className="fas fa-circle-notch"
  />
</div>
`;
